src/bundle/Resources/public/css/structure.css

```diff
@@ -0,0 +1,36 @@
+html {
+    height: 100%;
+}
+
+body {
+    min-height: 100%;
+    display: flex;
+}
+
+ez-platform-ui-app {
+    display: flex;
+    flex: 1 0 100%;
+    flex-direction: column;
+}
+
+ez-navigation-hub {
+    flex: 0 0;
+}
+
+main {
+    flex: 1 0;
+    display: flex;
+}
+
+#discovery {
+    flex: 0 0 230px;
```

Comment:
@SylvainGuittard @inakijv
this is based on PlatformUI 1.x layout so I guess this will be different, especially with eZ action button having icons on top of the text, but maybe this should also be responsive ?